packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-function-app.ts

```diff
@@ -27,7 +27,7 @@ export class TerraformFunctionApp {
       resourceGroupName: resourceGroupResource.name,
       servicePlanId: servicePlanResource.id,
       appSettings: {
-        WebPubSubConnectionString: webPubsubResource.primaryConnectionString,
+        WebPubSubConnectionString: webPubsubResource?.primaryConnectionString || '',
```

Comment:
Shouldn't we just leave this as it was so that `WebPubSubConnectionString` is undefined  (equivalent to not providing it) instead of the empty string?

Reply:
WebPubSubConnectionString doesn't accept undefined